docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,180 @@
+We will also only give the central device the keyboard name.
+
+For the dongle configuration we also want to specify that we are expecting 2 peripherals, one for the left half and one for the right half.
+We will also set the `ZMK_SLEEP` to `n` to prevent the dongle from going to sleep. 
+If the dongle is sleeping, keystrokes will not be sent to the host device, until the dongle wakes up.
+
+Finally, we will set the `BT_CLTR_TX_PWR_PLUS_8` to `y` to increase the transmit power of the dongle.
+Normally this would drain the battery faster, but since the dongle is connected to the host device, it will not be a problem.
+
+There is also an option to still use the left half as the central, and the right half as the peripheral. In that case the dongle will not be used.
+
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_BOARD_DONGLE
+
+config ZMK_KEYBOARD_NAME
+    default "My Board"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+    default y
+
+config ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS
+    default 2
+
+config ZMK_SLEEP
+    default n
+
+config BT_CLTR_TX_PWR_PLUS_8
+    default y
+
+endif
+
+if SHIELD_MY_BOARD_LEFT
+# Setting this to y will make the left half the central and the right half the peripheral
+config ZMK_SPLIT_ROLE_CENTRAL
+    default n
+
+if ZMK_SPLIT_ROLE_CENTRAL
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+endif # ZMK_SPLIT_ROLE_CENTRAL
+
+endif
+
+if SHIELD_MY_BOARD_DONGLE || SHIELD_MY_BOARD_LEFT || SHIELD_MY_BOARD_RIGHT
+
+config ZMK_SPLIT
+    default y
+
+endif
+```
+
+### Dongle Overlay
+
+In most common cases the dongle will not have any keys, in that case we can instead use a mock kscan module to simulate the keys.
+
+```dts title="my_board_dongle.overlay"
+
+/ {
+    chosen {
+        zmk,kscan = &mock_kscan;
+        // zmk,kscan = &kscan0;
+    };
+
+    mock_kscan: kscan_0 {
+        compatible = "zmk,kscan-mock";
+        columns = <0>;
+        rows = <0>;
+        events = <0>;
+    };
+};
+```
+
+### Building the firmware
+
+After writing the configuration files, you can modify the `build.yml` file to include the dongle configuration.
+
+Please keep in mind that the dongle does not have to be the same microcontroller as the keyboard halves.
+Any microcontroller can be used as the dongle that supports BLE.
+
+```yaml
+include:
+  - board: nice_nano_v2
+    shield: my_board_left
+  - board: nice_nano_v2
+    shield: my_board_right
+  - board: nice_nano_v2
+    shield: my_board_dongle
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+After creating your fresh new firmware files, 
+you can go ahead and flash the settings_reset firmware for each half of the keyboard (including the dongle).
+Flash the dongle firmware to the dongle and the left and right firmware to the respective halves.
+
+:::note
+More information about reseting split keyboards can be found under [Troubleshooting Connection Issues](../troubleshooting/connection-issues.mdx#reset-split-keyboard-procedure)
+:::
```

Comment:
I think all of these lines can be replaced with a :::warning that resetting is required prior to flashing this. I would assume that users know that left firmware gets flashed to the left half etc etc.

Reply:
Slimmed it down